_resources/port1.0/group/qt6_variables-1.0.tcl

```diff
@@ -118,38 +118,140 @@ namespace eval qt6pg {
     #
     ###############################################################################
     array set qt6_component_lib {
+        qt3d {
```

Comment:
do all the Qt modules that you added here build? If not, they probably shouldn't be added now as they will show up as subports for the qt6 port - correct?

Reply:
Yes they do.  But I forgot to remove qtlocation from the portfile, so will do that shortly.

Reply:
That last force push removed qtlocation.

Reply:
okay, sorry I might not have looked carefully enough... I just upgrade to Ventura so am building a lot of stuff before I can get to this PR to do some local testing.... Once I've done that I'll either merge this PR or get back here with comments. Thanks @jhoyt4 for working on this!